### PR TITLE
Testing pipeline updates

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Some (mostly minor) updates to various packing scripts and pipelines.

### CODE REVIEW

File: scripts/ci/validateassets.ps1 [hunk 1]
```diff
@@ -50,54 +50,54 @@ $Directory = Resolve-Path -Path $Directory
 #
 # The folders are relative to the RepoRoot.
 $packages = [ordered]@{
-    "StandardAssets" =  @(
+    "StandardAssets" = @(
         "MRTK/StandardAssets"
     );
-    "Foundation" =      @(
+    "Foundation"     = @(
```
Comment (RogPodge) [inline scripts/ci/validateassets.ps1:56]: Is this (and the other instances of this kind of spacing alignment) typical for powershell files? It's the first I've seen it and it looks kinda janky 😅

**Review by RogPodge — APPROVED** [review]
Looks fine, just not sure if the spacing stuff is intentional/expected.

File: scripts/ci/validateassets.ps1 (hunk 1 above)
Comment (keveleigh) [reply scripts/ci/validateassets.ps1:56]: i guess haha? it autoformatted and aligned with the furthest out name so the `=`s are vertically aligned. I can always revert this though! It's probably more personal preference

Comment (keveleigh) [reply scripts/ci/validateassets.ps1:56]: Looks like we were previously aligning the `@(`s, so this change also shifts the `=`s over

**Pull request merged**